src/app/inventory/RatingIcon.m.scss

```diff
@@ -1,7 +1,7 @@
 .godroll {
   font-size: 0.8em !important;
   margin-right: auto;
-  color: #0b486b;
+  color: var(--theme-item-polaroid-godroll);
```

Comment:
I did add this, then removed it since the trashrolls are always red which is legible in all themes atm so it felt like adding a variable for the sake of it. Doesn't seem like there's value in adding it until we need to?

Reply:
That's fair. If we don't plan on using them, perhaps we should remove the variables [here](https://github.com/guise/DIM/blob/b5e4658a80391cf6f019158a677fd05bdb76a372/src/app/_theme.scss#L69) and [here](https://github.com/guise/DIM/blob/b5e4658a80391cf6f019158a677fd05bdb76a372/src/app/_theme-dark.scss#L73)?